compositeCommon/AMMobility.pri

```diff
@@ -24,15 +24,18 @@ linux-g++-64 {
 	}
 }
 
-# Special build paths and options for running on the Jenkins auto-build server (currently at http://beamteam.usask.ca:8080)
+# Special build paths and options for running on the Jenkins auto-build server (currently at http://srv-aci-01.clsi.ca)
 CONFIG(jenkins_build) {
-		# Disable Qt Mobility Video until the Jenkins-machine supports that
-		CONFIG -= mobility
+	# Disable Qt Mobility Video until the Jenkins-machine supports that
+	message("Disabling Qt Mobility.")
```

Comment:
Are these messages used for debugging, or they a more permanent logging mechanism for the Jenkins server?
(ie. are they still needed?)